node-notes-cli/app.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+let dataParse;
+let dataStringify;
```

Comment:
You don't need to declare these here. You can do it inside your `readFile` callback.